test/integration/Deployment.t.sol

```diff
@@ -0,0 +1,413 @@
+        assertEq(WardsLike(ilkInst.stableSwapper).wards(facilitator2), 1);
+        assertEq(WardsLike(ilkInst.stableDepositorUniV3).wards(facilitator1), 1);
+        assertEq(WardsLike(ilkInst.stableDepositorUniV3).wards(facilitator2), 1);
+        assertEq(WardsLike(ilkInst.conduitMover).wards(facilitator1), 1);
+        assertEq(WardsLike(ilkInst.conduitMover).wards(facilitator2), 1);
+
+        assertEq(StableSwapper(ilkInst.stableSwapper).buds(stableSwapperKeeper1), 1);
+        assertEq(StableSwapper(ilkInst.stableSwapper).buds(stableSwapperKeeper2), 1);
+        assertEq(StableDepositorUniV3(ilkInst.stableDepositorUniV3).buds(stableDepositorUniV3Keeper1), 1);
+        assertEq(StableDepositorUniV3(ilkInst.stableDepositorUniV3).buds(stableDepositorUniV3Keeper2), 1);
+        assertEq(ConduitMover(ilkInst.conduitMover).buds(conduitMoverKeeper1), 1);
+        assertEq(ConduitMover(ilkInst.conduitMover).buds(conduitMoverKeeper2), 1);
+
+        assertEq(WardsLike(ilkInst.vault).wards(PAUSE_PROXY), 0);
+        assertEq(WardsLike(ilkInst.vault).wards(allocatorProxy), 1);
+
+        assertEq(WardsLike(ilkInst.buffer).wards(PAUSE_PROXY), 0);
+        assertEq(WardsLike(ilkInst.buffer).wards(allocatorProxy), 1);
+
+        assertEq(WardsLike(ilkInst.swapper).wards(PAUSE_PROXY), 0);
+        assertEq(WardsLike(ilkInst.swapper).wards(allocatorProxy), 1);
+
+        assertEq(WardsLike(ilkInst.depositorUniV3).wards(PAUSE_PROXY), 0);
+        assertEq(WardsLike(ilkInst.depositorUniV3).wards(allocatorProxy), 1);
+
+        assertEq(WardsLike(ilkInst.stableSwapper).wards(PAUSE_PROXY), 0);
+        assertEq(WardsLike(ilkInst.stableSwapper).wards(allocatorProxy), 1);
+
+        assertEq(WardsLike(ilkInst.stableDepositorUniV3).wards(PAUSE_PROXY), 0);
+        assertEq(WardsLike(ilkInst.stableDepositorUniV3).wards(allocatorProxy), 1);
+
+        assertEq(WardsLike(ilkInst.conduitMover).wards(PAUSE_PROXY), 0);
+        assertEq(WardsLike(ilkInst.conduitMover).wards(allocatorProxy), 1);
+
+        assertEq(ChainlogLike(LOG).getAddress("ILK_A_ALLOCATOR_VAULT"),  ilkInst.vault);
+        assertEq(ChainlogLike(LOG).getAddress("ILK_A_ALLOCATOR_BUFFER"), ilkInst.buffer);
+
+        assertEq(IlkRegistryLike(ILK_REGISTRY).count(),     previousIlkRegistryCount + 1);
+        assertEq(IlkRegistryLike(ILK_REGISTRY).pos(ILK),    previousIlkRegistryCount);
+        assertEq(IlkRegistryLike(ILK_REGISTRY).join(ILK),   address(0));
+        assertEq(IlkRegistryLike(ILK_REGISTRY).gem(ILK),    address(0));
+        assertEq(IlkRegistryLike(ILK_REGISTRY).dec(ILK),    0);
+        assertEq(IlkRegistryLike(ILK_REGISTRY).class(ILK),  5);
+        assertEq(IlkRegistryLike(ILK_REGISTRY).pip(ILK),    sharedInst.oracle);
+        assertEq(IlkRegistryLike(ILK_REGISTRY).xlip(ILK),   address(0));
+        assertEq(IlkRegistryLike(ILK_REGISTRY).name(ILK),   bytes32ToStr("ILK-A"));
+        assertEq(IlkRegistryLike(ILK_REGISTRY).symbol(ILK), bytes32ToStr("ILK-A"));
```

Comment:
Probably better to avoid using `bytes32ToStr` here so we can catch issues with its implementation (see previous comment)

```suggestion
        assertEq(IlkRegistryLike(ILK_REGISTRY).name(ILK),   string("ILK-A"));
        assertEq(IlkRegistryLike(ILK_REGISTRY).symbol(ILK), string("ILK-A"));
```

Reply:
Done in a separate commit.